src/components/utils/ControlledAlert/ControlledAlert.tsx

```diff
@@ -16,8 +16,11 @@ export interface ControlledAlertProps {
   /** When the alert is discardable, this is the callback that gets fired on discard */
   onClose: () => void;
 
+  /** The alert severity */
+  type?: 'success' | 'info' | 'warning' | 'error' | 'default';
+
   /** The style of the ControlledAlert */
-  variant?: 'success' | 'info' | 'warning' | 'error' | 'default';
+  variant?: 'default' | 'opaque';
```

Comment:
would a better name be `solid` instead of `default` here? What do you think?

Reply:
Sounds lovely!